eventstore/model.go

```diff
@@ -352,19 +316,15 @@ func (t *Txn) Commit() error {
 	if t.discarded || t.commited {
 		return errAlreadyDiscardedCommitedTxn
 	}
-
-	events, err := t.model.store.eventcodec.Create(t.actions)
+	events, node, err := t.model.store.eventcodec.Create(t.actions)
```

Comment:
The `EventCodec` now returns the encoded `Node` of all the actions done in this txn.
This `Node` is sent via the `notifyTxnEvents` which will be received by `StoreThread`, which will `AddRecord` to the peer local Log.